docs/rfcs/merger_matcher/README.md

```diff
@@ -36,22 +37,33 @@ This document has notes on the proposed architecture.
 
 A matching phase is introduced to determine where works need to be combined.
 
-The input to this phase is the source identity for a work and a list of the identities that make up the combined work.
-The output is sets of identifiers of affected works that will be combined downstream by the merger
+The input to this phase is a work. A transformed work contains the work identifier, the work version and a list of identifiers of other works that the current work directly links to.
+The output is sets of identifiers of affected works alongside their versions. The merger will use the identifiers to read the works from the recorder VHS and it will use the versions in
+the message to ensure that the versions are still the most up to date ones. If so, it will combine the works. If not, it will discard the message.
 
 Example:
 ```[json]
     {
-        "linked-works": [
+       "linked-works-sets" : [
             {
-                "identifiers": [
-                    "sierra-system-number/b1234567",
-                    "miro-image-number/V003456"
+                "linked-works": [
+                    {
+                      "identifier": "sierra-system-number/b1234567",
+                      "version": 2
+                    },
+                    {
+                      "identifier": "miro-image-number/V003456",
+                      "version": 1
+                    },
+
                 ]
```

Comment:
Perhaps explicitly add how the linked work versions are obtained, and that we will drop if a linked work has not been seen yet.